src/components/SignUp.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import UserName from "./Username";
+import Email from "./Email";
+import Password from "./Password";
+
+function SignUp() {
+  return(
+    <div>
+      <div className="flex flex-col items-center justify-center py-4 pt-12">
+        <p className="text-5xl font-light">Sign Up</p>
+        <p className="text-[#5cb85c] pt-2 text-lg my-2">Have an account?</p>
+      </div>
+
+      <div className= "flex items-center justify-center px-4">
+        <div className="flex flex-col w-[640px] gap-y-5">
+          <UserName />
```

Comment:
All these components look pretty similar inside. Lets merge them all into 1 component that accepts parameters to change how it works.

Reply:
Referring to Username, email and password.

Reply:
Addressed. `Username`, `Email`, `Password` components have been combined as a re-usuable 'Form' component.  Please see my updated codes. Thanks!